docs/balances/FAQs.md

```diff
@@ -8,6 +8,50 @@ tags: [Balances]
 
 Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.
 
+Valuations are created at approximately the same time every day, for the previous day. Transactions booked with a date of `t` are only reflected in the valuation for `t+1`.
+
+The balance is not dated, and is continuously updated, and will therefore reflect any actions taken on the portfolio, such as deposits, orders, or bonuses, as they happen.
+
+### Example
+
+```mermaid
+gantt
+dateFormat YYYY-MM-DD HH:mm
+axisFormat %m-%d %H:%M
+title Dated valuations
+todayMarker off
+
+    section Valuation
+    Latest Valuation 0GBP :active, val1, 2024-02-19 17:00, 13h
```

Comment:
I'd be explicit with the valuation date. "Latest" is only available internally.

Reply:
Also timestamps give the impression "Latest valuation" lives from 2024-02-19 17:00 -> 2024-02-20 06:00 when in reality it is two valuation intervals.